src/Raven.Server/ServerWide/Context/FilesOperationContext.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using Raven.Server.Files;
+using Voron;
+
+namespace Raven.Server.ServerWide.Context
+{
+    public class FilesOperationContext : TransactionOperationContext<FilesTransaction>
+    {
+        private readonly FileSystem _fileSystem;
+
+        public short TransactionMarkerOffset;
+
+        private short _currentTxMarker;
+
+        public short GetTransactionMarker()
+        {
```

Comment:
Code duplication: this method is exactly the same as in `DocumentsOperationContext`